README.md

```diff
@@ -31,13 +31,27 @@ Smartmi Air Purifier | zhimi.airpurifier.za1 |  | [zhimi.airpurifier.za1](config
 
 Some of the devices have more than one model (like Mi Air Purifier 3C). If their MIoT specifications are compatible, the ESPHome config will be usable with all of them.
 
-## Not supported devices
+## Unsupported devices
 
-Some devices have Xiaomi proprietary chip instead of ESP32 chip. Those devices are not supported.
+- Devices using only the ESP microcontroller to directly control the hardware.
+  
+  For these, this component will not work, but you can create an ESPHome 
+  configuration directly;
```

Comment:
I think I know what you mean, but to me it's more confusing than helping :P
I'll reword that

Reply:
Done, but I'm open to any suggestion to improve my version ;)